src/Move.java

```diff
@@ -0,0 +1,17 @@
+public class Move {
+    private Pawn pawn;
```

Comment:
What if we represent a pawn here by its index (you can think of it like an "id")? Then the `Player` and `PlayerTest` don't need to change at all. 

#### Why?
The `Player` is responsible for its associated pawns. It's typically a "bad idea" to allow a third class to have an _ownership_ reference, like a property, to another class' associations (in this case, pawns). It complicates the relationship of classes to one another. The connections will start to look like a spiderweb, and the code will be more confusing.

However, It's not always a "bad idea" for another class to have a _transient_ reference to another's associations, like a local variable. So we can still get access to pawns and use them when we need to, as long as our use it temporary.